packages/mdc-form-field/README.md

```diff
@@ -7,141 +7,95 @@ path: /catalog/input-controls/form-fields/
-```
-
-#### CommonJS
-
-```javascript
-const mdcFormField = require('mdc-form-field');
-const MDCFormField = mdcFormField.MDCFormField;
-const MDCFormFieldFoundation = mdcFormField.MDCFormFieldFoundation;
-```
-
-#### AMD
-
-```javascript
-require(['path/to/mdc-form-field'], mdcFormField => {
-  const MDCFormField = mdcFormField.MDCFormField;
-  const MDCFormFieldFoundation = mdcFormField.MDCFormFieldFoundation;
-});
-```
-
-#### Global
+> _NOTE_: MDC Form Field is **not** intended for cases where a label and input are already handled together in a component's styles and logic. For example, [MDC Text Field](../mdc-text-field) already manages a label and input together under its own root element.
 
-```javascript
-const MDCFormField = mdc.formField.MDCFormField;
-const MDCFormFieldFoundation = mdc.formField.MDCFormFieldFoundation;
-```
+### JavaScript Instantiation
 
-### Instantiation
+If you are using MDC Form Field with an MDC Web component that has a [ripple](../mdc-ripple) effect, you can instantiate `MDCFormField` and set its [`input` property](#MDCFormField-properties-and-methods) to activate the ripple effect upon interacting with the label. Here is an example using [MDC Radio](../mdc-radio):
 
-```javascript
+```js
 import {MDCFormField} from '@material/form-field';
+import {MDCFormField} from '@material/radio';
 
 const formField = new MDCFormField(document.querySelector('.mdc-form-field'));
+const radio = new MDCRadio(document.querySelector('.mdc-radio'));
+formField.input = radio;
 ```
 
-### MDCFormField API
+> See [Importing the JS component](../../docs/importing-js.md) for more information on how to import JavaScript.
 
-The `MDCFormField` functionality is exposed through a single accessor method.
+## Variants
 
-#### MDCFormField.input
+### Label position
 
-Read-write property that works with an instance of an MDC Web input element.
+By default, the input will be positioned before the label. You can position the input after the label by adding the `mdc-form-field--align-end` class:
 
-In order for the label ripple integration to work correctly, this property needs to be set to a
-valid instance of an MDC Web input element which exposes a `ripple` getter.
+```html
+<div class="mdc-form-field mdc-form-field--align-end">
+  <input type="checkbox" id="input">
+  <label for="input">Label</label>
+</div>
+```
 
-```javascript
-const formField = new MDCFormField(document.querySelector('.mdc-form-field'));
-const radio = new MDCRadio(document.querySelector('.mdc-radio'));
+## `MDCFormField` Properties and Methods
 
-formField.input = radio;
-```
+Property | Value Type | Description
+--- | --- | ---
+`input` | String | Gets and sets the form field input. 
+
+In order for the label ripple integration to work correctly, this property needs to be set to a valid instance of an MDC Web input element which exposes a `ripple` getter. No action is taken if the `input` property is not set or the input instance doesn't expose a `ripple` getter.
```

Comment:
"this property" -> "the `input` property" (this sentence used to be in the context of its heading but is now in the context of the table above)